frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { ConfirmationModal } from '../../../components/ConfirmationModal'
+import { handleCreateBooking } from '../../../store/facilityBooking/action'
+import { Booking } from '../../../store/facilityBooking/types'
+
+export default function ConflictBookingModal(props: { booking: Booking | undefined }) {
+  const [open, setOpen] = useState(false)
+  /* TODO implement function to toggle state to true */
+  function changeStatus() {
+    setOpen(!prevOpen)
+  }
+  function onRightClick() {
+    /*TODO to update createbooking function in store instead*/
+    changeStatus()
+    handleCreateBooking(props.booking?.bookingID ? true : false)
+  }
+
+  return (
+    <ConfirmationModal
+      isModalOpen={open}
+      rightButtonType="primary"
+      rightButtonText="Confirm"
+      title="Conflict in your bookings!"
+      onLeftButtonClick={changeStatus}
+      onRightButtonClick={onRightClick}
+      leftButtonType="secondary"
+      leftButtonText="Cancel"
+      hasLeftButton
+      description="Do you still want to continue with the conflicting bookings?"
+      link="View Conflicts"
```

Comment:
requires text for link and actual link to call when it is clicked

Reply:
Resolved in commit 509ab25176f49a4a836ede036eb3559e1af54833.

Reply:
use camel case! onLinkClick instead of onlinkClick